apps/prow/deploy.sh

```diff
@@ -57,4 +59,4 @@ make update-plugins
 echo "Update k8s-infra-prow prowjobs"
 make update-prowjobs
 
-kubectl --context="${context}" --namespace="${namespace}" apply -Rf cluster/
+kubectl --context="${context}" apply -Rf cluster/
```

Comment:
I'm wary of this because the thing applying these changes has rights to more than just the namespaces relevant to prow.  I don't want namespaces to magically get created from files that sneak in here.  Is it possible to segment resources by namespace into separate dirs?

Reply:
I'll move the namespace to a separate dir.